### PR TITLE
Quadratic complexity test

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

File: test/stSolidityTestFiller.json [hunk 1]
```diff
@@ -506,5 +495,61 @@
...
+		"//" : "}									",
+                "code" : "0x60003560e060020a9004806361a4770614601557005b601e6004356024565b60006000f35b60008160008190555073b94f5374fce5edbc8e2a8697c15331677e6ebf0b90505b600082131560bf5780600160a060020a03166000600060007f6a7573740000000000000000000000000000000000000000000000000000000081526004017f63616c6c000000000000000000000000000000000000000000000000000000008152602001600060008560155a03f150506001820391506045565b505056",
+                "nonce" : "0",
+                "storage" : {
+                }
+            },
+
+	    "b94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "100000",
+                "code" : "{ (CALLDATACOPY 0 0 32) (SSTORE 0 (MLOAD 0)) }",
```
Comment (CJentzsch) [inline test/stSolidityTestFiller.json]: you can erase this line

File: test/stSolidityTestFiller.json [hunk 2]
```diff
@@ -506,5 +495,61 @@
...
+                "nonce" : "0",
+                "storage" : {
+                }
+            }
+        },
+        "transaction" :
+        {
+	    "//" : "run(int256)",
+            "data" : "0x61a47706000000000000000000000000000000000000000000000000000000000000c350",
+	    "gasLimit" : "904+68*x+e",
```
Comment (CJentzsch) [inline test/stSolidityTestFiller.json]: wrong indenting

File: test/stSolidityTestFiller.json [hunk 3]
```diff
@@ -489,6 +461,23 @@
...
+		"//" : "				",
+		"//" : "contract contract2		",
+		"//" : "{				",
+		"//" : "    uint value2;		",
+		"//" : "    function run()		",
+		"//" : "    {				",
+		"//" : "        value2 = 335;		",
+		"//" : "    }				",
+		"//" : "}				",
                 "code" : "0x60003560e060020a90048063c040622614601557005b601b6021565b60006000f35b61014f60008190555056",
```
Comment (CJentzsch) [inline test/stSolidityTestFiller.json]: When you have two contracts, which code did you put in?  And does the other contract in any way affect the "code"

Comment (winsvega) [inline test/stSolidityTestFiller.json]: Both contract's code is added here. 
Solidity allows to have them as 'class' objects in code. 
But what curious is which method of the contract would be called if they are exactly the same

**Pull request merged**